acknowledging.html

```diff
@@ -191,7 +191,7 @@
                   Unbehaun, Tim and
                   Wong, Samantha and
                   Yu, Pei},
-  title        = {Gammapy: Python toolbox for gamma-ray astronomy},
+  title        = {Gammapy: v1.3, https://doi.org/10.5281/zenodo.14760974},
```

Comment:
Yes this works in A&A Bibtex format. 
One could also apply this formatting for previous version (v1.0, 1.1, 1.2)
By the way should we provide on the webpage one line for the different version ?

People publishing now may have started their work with older version and want to cite the old ones.
Like adding 2 extra green buttons for older versions ?